### DISCUSSION TITLE
New development policy

### DISCUSSION THREAD

**Discussion by boomzero** [XMOJ-Script-dev/XMOJ-Script#727] [Announcements]
To increase activity in our team and to encourage developers to contribute to this project, I have decided to set the following rule:

All members of the dev team must at least fix (not close) one issue every month (not including break periods). Should they fail to do so, they will be removed from the team. They can rejoin at any time, but they will have to go through the same process as everyone else.

Thank you.

@ptsq-bot @PythonSmall-Q 

Note: obviously doesn’t apply to @langningchen , he is here to provide tech support(

**Comment by boomzero** [comment]
Perfectly reasonable.

**Comment by PythonSmall-Q** [comment]
@boomzero 对我来说不可行，主要原因在于我妈不让我上电脑

**Reply by boomzero** [reply]
Once a month? If you aren’t here to work what are you here for?

**Comment by boomzero** [comment]
@PythonSmall-Q 
![image](https://github.com/user-attachments/assets/d70aaa7c-9416-47b1-a0a4-b43fb18e7301)

**Reply by boomzero** [reply]
Doing homework

**Comment by boomzero** [comment]
Note: this should take effect today.
